google/cloud/bigtable/row_filters.py

```diff
@@ -481,7 +481,7 @@ def to_dict(self) -> dict[str, bytes]:
         return {"value_regex_filter": self.regex}
 
 
-class ExactValueFilter(ValueRegexFilter):
+class LiteralValueFilter(ValueRegexFilter):
```

Comment:
Do we need to go through deprecation process, to let users know that `ExactValueFilter` will be removed in v3, and that they should be using `LiteralValueFilter` instead?

Reply:
It will not be removed right away, since there is a copy of `ExactValueFilter` and all other classes from the existing library in `google.cloud.bigtable.deprecated` in the `v3` branch

The `v3` deprecation process will be a bit tricky, but we will make sure that all existing classes are accessible for at least a decent transition period